setup.py

```diff
@@ -49,6 +49,10 @@ def build_extension(self, ext):
             "-DCMAKE_BUILD_TYPE={}".format(cfg),  # not used on MSVC, but no harm
         ]
         build_args = []
+        # Adding CMake arguments set as environment variable
+        # (needed e.g. to build for ARM OSx on conda-forge)
+        if "CMAKE_ARGS" in os.environ:
+            cmake_args += os.environ["CMAKE_ARGS"].split(" ")
```

Comment:
Thanks! Just a suggestion:
```suggestion
            cmake_args += list(filter(None, os.environ["CMAKE_ARGS"].split(" ")))
```
I was just debugging somewhere, and I find that this can make things slightly more robust

Reply:
```suggestion
            cmake_args += [item for item in os.environ["CMAKE_ARGS"].split(" ") if item]
```
Perhaps? `list(filter(None` is a little hard to read, IMO.

Reply:
Also fine !